src/GraphQL.Tests/GraphQL.Tests.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="../Tests.props" />
   
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;netstandard2.0</TargetFrameworks>
```

Comment:
But can tests work on netstandard?

Reply:
Will be undoing this and taking a diff approach